### PR TITLE
Use "Bleeding Edge" and "Tagged Beta"

### PR DESCRIPTION
**Pull request by kraftbj**
Instead of calling it Beta and Release Candidates when all of our internal language calls it "master" (or "master-stable") and "Beta", let's update this.

It gets confusing to have master-stable referred to as Beta, then talk about "Beta 1", which is actually a RC by the beta plugin language.

### CODE REVIEW

**Comment by samhotchkiss** [comment]
I'm good with this...

**Comment by kraftbj** [comment]
@zinigor said he likes this better and @csonnek concurred, so two HEs and two devs. Let's ship.

**Comment by samhotchkiss** [comment]
Just merging it doesn't ship it as an update.  Is this worth pushing an
update for?

**Comment by kraftbj** [comment]
@samhotchkiss It makes sense (to me) to bump the plugin version once a JP release cycle, assuming anything was changed and nothing critical requires a push mid-cycle.

tl;dr: Let's update, unless there's anything else we can slip in real quick like.
